x-pack/plugins/infra/server/types.ts

```diff
@@ -27,6 +28,7 @@ export interface InfraPluginSetup {
 
 export interface InfraPluginStart {
   logViews: LogViewsServiceStart;
+  getMetricIndices: ReturnType<typeof makeGetMetricIndices>;
```

Comment:
Conditional types like this can break the automated API doc generation. How about we define an explicit type for the getter?

Reply:
Good to know!